Modules/EthereumClassicMainModule.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types = 1);
+
+/*  Idea (c) 2023 Nikita Zhavoronkov, nikzh@nikzh.com
+ *  Copyright (c) 2023 3xpl developers, 3@3xpl.com, see CONTRIBUTORS.md
+ *  Distributed under the MIT software license, see LICENSE.md  */
+
+/*  This is the main Ethereum Classic module. It requires either a geth node to run. Please note that `status` is
+ *  not available on geth for some older transactions when requesting receipts and a special fix is required.  */
+
+final class EthereumClassicMainModule extends EVMMainModule implements Module
+{
+    function initialize()
+    {
+        // CoreModule
+        $this->blockchain = 'ethereum-classic';
+        $this->module = 'ethereum-classic-main';
+        $this->is_main = true;
+        $this->first_block_date = '2015-07-30';
+        $this->currency = 'ethereum-classic';
+        $this->currency_details = ['name' => 'Ethereum Classic', 'symbol' => 'ETC', 'decimals' => 18, 'description' => null];
+
+        // EVMMainModule
+        $this->evm_implementation = EVMImplementation::geth;
+        $this->extra_features = [EVMSpecialFeatures::HasOrHadUncles];
+        $this->reward_function = function($block_id)
+        {
+            // https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1017.md
```

Comment:
Should be replaced with the actual link https://ecips.ethereumclassic.org/ECIPs/ecip-1017

Reply:
Fixed in [7483500](https://github.com/3xplcom/Core/pull/67/commits/74835006bd39f879bb6787d965759901383726b1).